x-pack/plugins/observability_solution/synthetics/public/apps/synthetics/components/settings/data_retention/unprivileged.tsx

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import { FormattedMessage } from '@kbn/i18n-react';
+import { EuiEmptyPrompt, EuiIcon, EuiLink, EuiMarkdownFormat } from '@elastic/eui';
+import React from 'react';
+import { i18n } from '@kbn/i18n';
+import { css } from '@emotion/react';
+import { useKibana } from '@kbn/kibana-react-plugin/public';
+
+export const Unprivileged = ({ showIlmMessage }: { showIlmMessage?: boolean }) => {
+  const {
+    services: { docLinks },
+  } = useKibana();
+  return (
+    <EuiEmptyPrompt
+      data-test-subj="syntheticsUnprivileged"
+      color="plain"
+      icon={<EuiIcon type="logoObservability" size="xl" />}
+      title={
+        <h2>
+          <FormattedMessage
+            id="xpack.synthetics.dataRetention.unprivileged.unprivilegedTitle"
+            defaultMessage="Missing privileges"
+          />
+        </h2>
+      }
+      body={
+        <p>
+          <FormattedMessage
+            id="xpack.synthetics.params.unprivileged.unprivilegedDescription"
+            defaultMessage="You need additional privileges to view Synthetics app data usage and retention settings. {docsLink}"
+            values={{
+              docsLink: (
+                <EuiLink
+                  data-test-subj="syntheticsUnprivilegedLearnMoreLink"
+                  href={docLinks?.links.synthetics.featureRoles}
+                  target="_blank"
+                >
+                  {i18n.translate('xpack.synthetics.monitorManagement.projectDelete.docsLink', {
+                    defaultMessage: 'Learn more',
+                  })}
+                </EuiLink>
+              ),
+            }}
+          />
+        </p>
+      }
+      footer={
+        <EuiMarkdownFormat
+          css={css`
+            text-align: initial;
+          `}
+          children={`\n- ${INDEX_PRIVILEGES} ${
+            showIlmMessage === false ? '' : ` \n- ${CLUSTER_PRIVILEGES}`
```

Comment:
When `showIlmMessage` is **undefined** this will use the fallback value. Is it what you want? Otherwise, you could set the default value to `false` in the variable definition on line 14.

```
{showIlmMessage = false}: ...
```

Reply:
I guess a better name for the prop is `hideIlmMessage` and make it a positive flag.

So, the usage would be `hideIlmMessage === true, do not display; display by default`.

We're overriding the original functionality of this component to not show anything ILM-related in Serverless, because there is no ILM in Serverless. I'll change the name and use a default as you recommended.

Reply:
da71ba5ab5d0a61b91a610da526b044180d56644